exercises/word-count/src/example.c

```diff
@@ -50,8 +50,17 @@ int word_count(char *input_text, word_count_word_t words[MAX_WORDS])
 
       // add if not yet counted else increment count.
       if (-1 == index) {
+         // reject anything that would overrun buffers
+         if (MAX_WORD_LENGTH < strlen(test_word)) {
+            unique_words = EXCESSIVE_LENGTH_WORD;
+            break;
+         }
+         if (MAX_WORDS == unique_words) {
+            unique_words = EXCESSIVE_NUMBER_OF_WORDS;
+            break;
+         }
```

Comment:
Important lesson for me in why some coding standards require one exit from a function.  Needed to clean up after myself.

Reply:
Good catch, that's why if possible I usually just use Cpp and [RAII](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rr-raii). In C, a lot of code ends up using `goto exit;` to cleanup resources if the control flow is too complicated, which always looks messy to me.

Reply:
I didn't catch it, valgrind did.